torch/csrc/utils/tensor_layouts.cpp

```diff
@@ -1,38 +1,51 @@
-#include <torch/csrc/utils/tensor_layouts.h>
 #include <ATen/Layout.h>
 #include <c10/core/ScalarType.h>
 #include <torch/csrc/DynamicTypes.h>
 #include <torch/csrc/Exceptions.h>
 #include <torch/csrc/Layout.h>
 #include <torch/csrc/python_headers.h>
 #include <torch/csrc/utils/object_ptr.h>
+#include <torch/csrc/utils/tensor_layouts.h>
 
-namespace torch { namespace utils {
+namespace torch {
+namespace utils {
 
 void initializeLayouts() {
   auto torch_module = THPObjectPtr(PyImport_ImportModule("torch"));
-  if (!torch_module) throw python_error();
+  if (!torch_module)
+    throw python_error();
 
-  PyObject *strided_layout = THPLayout_New(at::Layout::Strided, "torch.strided");
+  PyObject* strided_layout =
+      THPLayout_New(at::Layout::Strided, "torch.strided");
   Py_INCREF(strided_layout);
   if (PyModule_AddObject(torch_module, "strided", strided_layout) != 0) {
     throw python_error();
   }
   registerLayoutObject((THPLayout*)strided_layout, at::Layout::Strided);
 
-  PyObject *sparse_coo_layout = THPLayout_New(at::Layout::Sparse, "torch.sparse_coo");
+  PyObject* sparse_coo_layout =
+      THPLayout_New(at::Layout::Sparse, "torch.sparse_coo");
```

Comment:
i do not like spurious whitespace changes, sam i am

Reply:
The clang tests would not pass without this. However I'll revert this back to the original.